### DISCUSSION TITLE
conexao banco de dados zim usando ODBC <zim database connection using ODBC>

### DISCUSSION THREAD

**Discussion by sisazim** [dbeaver/dbeaver#13459] [Q&A] [answered]
OLA.
drive odbc instalado e funcionando no windows-10 e acesso excel funciona. Mas o DBeaver nao conecta. Tem como enviar um manual de como configurar a seleçao do drive ja instalado? 

< HELLO.
odbc drive up and running on windows-10 and excel access works. But DBeaver does not connect. Is it possible to send a manual on how to configure the selection of the drive already installed? >
![odbc](https://user-images.githubusercontent.com/20546226/128190967-522e6722-0cfd-4af6-a532-4fb814eac6dc.jpg)
![odbc1](https://user-images.githubusercontent.com/20546226/128191278-68f36585-3af8-449c-aa66-3423f0238c73.jpg)

**Comment by uslss** [comment, marked as answer]
Unfortunately, dbeaver supports only 64 bit driver version
